eth/api_backend.go

```diff
@@ -267,11 +267,11 @@ func (b *EthAPIBackend) SubscribeLogsEvent(ch chan<- []*types.Log) event.Subscri
 func (b *EthAPIBackend) SendTx(ctx context.Context, signedTx *types.Transaction) error {
 	if vm.UsingOVM {
 		to := signedTx.To()
-		if to != nil && bytes.Equal(to.Bytes(), common.Address{}.Bytes()) {
+		if to != nil && *to == (common.Address{}) {
```

Comment:
Are the parens needed?

```suggestion
		if to != nil && *to == common.Address{} {
```

Reply:
If I remove the parens, it doesn't compile